default.env

```diff
@@ -3,7 +3,7 @@ WORDPRESS_DB_HOST=db:3306
 WORDPRESS_DB_USER=wordpress
 WORDPRESS_DB_PASSWORD=wordpress
 WORDPRESS_DB_NAME=wordpress
-WORDPRESS_DEBUG=1
+WORDPRESS_DEBUG=false
```

Comment:
May I know why we need to change WORDPRESS_DEBUG value?

Reply:
Reverted back, this was done for testing as I though some PHP notification might be breaking puppeteer.

Reply:
> I hand no issue running `npm run prepare` on my local without #2567 fix. I did test in form of Draft PR #2568 to exclude specifics of my local I added `npm run prepare` as one of the jobs and it fished it without problems

Thanks for creating a draft PR #2568 @bartech. I have another test for this PR by doing `git pull` and run `npm run prepare` in my local. Although the command is successful, I still can see the error like below : 
```
make: *** [Release/obj.target/fse/fsevents.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/luthfibintoro/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Darwin 21.5.0
gyp ERR! command "/Users/luthfibintoro/.nvm/versions/node/v10.16.0/bin/node" "/Users/luthfibintoro/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" "--module_name=fse" "--module_path=/Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64" "--napi_version=4" "--node_abi_napi=napi"
gyp ERR! cwd /Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents
gyp ERR! node -v v10.16.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/Users/luthfibintoro/.nvm/versions/node/v10.16.0/bin/node /Users/luthfibintoro/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node --module_name=fse --module_path=/Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64 --napi_version=4 --node_abi_napi=napi' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:982:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
node-pre-gyp ERR! System Darwin 21.5.0
node-pre-gyp ERR! command "/Users/luthfibintoro/.nvm/versions/node/v10.16.0/bin/node" "/Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents/node_modules/node-pre-gyp/bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /Users/luthfibintoro/a8c/woocommerce-services/wp-calypso/node_modules/jest-haste-map/node_modules/fsevents
node-pre-gyp ERR! node -v v10.16.0
node-pre-gyp ERR! node-pre-gyp -v v0.10.3
node-pre-gyp ERR! not ok 
```
Do you mind to try to test this PR in your local to confirm if it doesn't produce the error above?